components/raftstore/src/store/peer.rs

```diff
@@ -1765,6 +1781,14 @@ where
                 self.last_applying_idx,
                 self.term(),
                 self.raft_group.store().region(),
+                |region| {
+                    util::find_sibling_regions(
+                        &ctx.store_meta,
+                        &ctx.cfg,
+                        &ctx.cop_cfg.value(),
+                        region,
+                    )
+                },
```

Comment:
Actually, this closure is useless because cmd can be proposed only if the peer is a leader. It's not a leader due to snapshot. 🤣 But I think it's ok for simplicity.